gateway/roles-mappings.yaml

```diff
@@ -29,4 +29,20 @@
 #        - ROLE_USER
 #      '[ROLE_GP.GDI.ADMINISTRATOR]':
 #        - ROLE_ADMINISTRATOR
-
+ 
+georchestra:
+  gateway:
+    roles-mappings:
+      '[ROLE_GP.GDI.*]':
```

Comment:
cc @pmauduit this reads like DT specifics ...

Reply:
indeed, this is a feature needed by DT, but we can probably leave it commented here